HW37Person2.0Localization/Person2.0/View/Custom/PersonButton.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  PersonButton.swift
+//  HW37Person2.0Localization
+//
+//  Created by Вячеслав Башур on 20/10/2024.
+//
+
+import Foundation
+import UIKit
+
+class PersonButton: UIButton {
+    enum Style {
+        case edit
+        case delete
+        case add
+        case close
+        case replace
+        case save
+    }
+    
+    enum Icon {
+        static let edit = UIImage(
+            systemName: "pencil.circle.fill"
+        )
```

Comment:
тут в целом можно было все в одну строку писать)  типа
```suggestion
        static let edit = UIImage(systemName: "pencil.circle.fill")
```